OpenSim/Simulation/Model/ConditionalPathPoint.h

```diff
@@ -50,6 +50,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ConditionalPathPoint, PathPoint);
         "The minimum and maximum values that the coordinate can range between, "
         "for which the PathPoint is active. Angular coordinates in radians.");
 
+//==============================================================================
+// CONNECTORS
+//==============================================================================
+    OpenSim_DECLARE_CONNECTOR(coordinate, Coordinate,
+        "The coordinate whose value this path point depends on.");
```

Comment:
What about:

```
"This PathPoint is active when this coordinate's value is within the range specified in the range property."
```
?

I don't want to duplicate the comment of the `range` property too much.